include/task_wdt/task_wdt.h

```diff
@@ -46,6 +46,9 @@ typedef void (*task_wdt_callback_t)(int channel_id, void *user_data);
  *               Pass NULL if no hardware watchdog fallback is desired.
  *
  * @retval 0 If successful.
+ * @retval Negative If the fallback hw_wdt is used and the install timeout API
+ *                  fails.  See wdt_install_timeout() API for possible return
+ *                  values.
  * @retval -ENOTSUP If assigning a hardware watchdog is not supported.
```

Comment:
suggestion:

```suggestion
 * @retval -ENOTSUP If assigning a hardware watchdog is not supported.
 * @retval -Errno Other negative errno if the fallback hw_wdt is used and the install timeout API
 *                  fails.  See wdt_install_timeout() API for possible return
 *                  values.
```

Reply:
No worries, updated to your suggestion.  One thing I'm concerned about is that `wdt_install_timeout()` also returns `-ENOTSUP` so there's an overlap.  Is this acceptable? or can anyone suggest a good course of action for this?